src/host/getset.cpp

```diff
@@ -359,29 +359,15 @@ void ApiRoutines::GetNumberOfConsoleMouseButtonsImpl(ULONG& buttons) noexcept
             WI_ClearFlag(gci.Flags, CONSOLE_USE_PRIVATE_FLAGS);
         }
 
-        const auto oldMouseMode{ WI_IsFlagSet(context.InputMode, ENABLE_MOUSE_INPUT) };
-        const auto newMouseMode{ WI_IsFlagSet(mode, ENABLE_MOUSE_INPUT) };
         const auto newQuickEditMode{ WI_IsFlagSet(gci.Flags, ENABLE_QUICK_EDIT_MODE) };

```

Comment:
we chatted about this checking the CONSOLE_QUICK_EDIT variation :)

Reply:
Yep! Changed